Corona-Warn-App/src/main/java/de/rki/coronawarnapp/util/permission/CameraPermissionSettings.kt

```diff
@@ -0,0 +1,25 @@
+package de.rki.coronawarnapp.util.permission
+
+import android.content.Context
+import de.rki.coronawarnapp.util.di.AppContext
+import de.rki.coronawarnapp.util.preferences.createFlowPreference
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class CameraPermissionSettings @Inject constructor(
+    @AppContext private val context: Context
+) {
+    private val prefs by lazy {
+        context.getSharedPreferences("camera.permission.settings", Context.MODE_PRIVATE)
```

Comment:
Should we use a "permission" specific setting, there may be more permission related settings in the future :thinking: 
Maybe we should just call it "CameraSettings"?

Reply:
Can you please elaborate more ? The class is called CameraSettings and the string is called camera.permission.settings? What should be exactly the change?

Reply:
I don't thin that Camera settings is suitable here. We don't have a camera feature to have a settings for it like instagram for example. We just doing this for permission and I think reflect what it does

Reply:
Just thinking out loud whether this needs a single preference. The file "camera.permission.settings" would be a "not nice" name for anything, whether we can give this any re-use potential as "CameraSettings" or "PermissionSettings" :thinking: